src/In.ProjectEKA.TMHHip/DataFlow/CustomWebclient.cs

```diff
@@ -0,0 +1,34 @@
+namespace In.ProjectEKA.TMHHip.DataFlow
+{
+    using System;
+    using System.Net;
+    using System.Security;
+
+    internal class CustomWebclient : WebClient
+    {
+        [SecuritySafeCritical]
+        private CustomWebclient() : base()
```

Comment:
Codacy found an issue: [Remove this redundant 'base()' call.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4940156)